### PR TITLE
Add new organizations to 12 step

### LINKED ISSUE #125: add the following to the 12-step popover
Double Trouble in Recovery- A recovery group for people recovering from mental
illness and substance abuse.

Participation Station
869 Sparta Court                      
Lexington, KY
Thursdays 12-1:00 p.m.
Sunday 3-4:00 p.m.
For more information, please contact (859) 492-6997

The Community Inn                                    
824 Winchester Road
Lexington, KY
Thursdays 7:30-8:30 p.m.
For more information, please contact (859) 312-0707

Eastern State Hospital                                                      
627 West Fourth Street
Lexington, KY
Participation in the meeting is limited to current hospital patients only

### PR DESCRIPTION
**Pull request by nikolaiwarner**
fixes #125

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -303,6 +315,12 @@ <h4 class="modal-title" id="twelveStepLabel">TWELVE STEP PROGRAMS</span></h4>
 
             <h3><strong>Pennyrile Area (Paducah & Hopkinsville)</strong></h3>
             <p><a href="http://www.na-pana.org">http://www.na-pana.org</a></p>
+            
+            <h3><strong>Participation Station</strong></h3>
+            <p>869 Sparta Court, Lexington, KY<br>
+            Thursdays 12-1:00 p.m.<br>
+            Sunday 3-4:00 p.m.<br>
+            Phone: <a href="tel:8594926997">(859) 492-6997</a></p>
```
Comment (chaserx) [inline index.html]: all three of these locations are where one can find `Double Trouble in Recovery` programs.

Comment (nikolaiwarner) [reply index.html]: Ok!

**Comment by eeeschwartz** [comment]
+1

**Comment by chaserx** [comment]
+1

**Pull request merged**